daemon/k8s_watcher.go

```diff
@@ -369,11 +388,15 @@ func (d *Daemon) deleteK8sNetworkPolicy(obj interface{}) {
 	}
 }
 
-// addK8sNetworkPolicyDeprecated FIXME remove in k8s 1.8
+// addK8sNetworkPolicyDeprecated
+// FIXME remove when we drop support to k8s Network Policy extensions/v1beta1
 func (d *Daemon) addK8sNetworkPolicyDeprecated(obj interface{}) {
```

Comment:
Would it be clearer if these functions were name `addK8sNetworkPolicyv1beta1` etc. for when we add support for for a `v2beta1` (or whatever).  I imagine we'd need to retain backwards compatibility anyway so while K8s moves on we must continue to work with older ones  and newer ones :/

Reply:
Fixed! I've done for every function